test/nerf/test_camera_utils.py

```diff
@@ -7,15 +8,46 @@
 from kornia.nerf.camera_utils import create_spiral_path, parse_colmap_output
 from kornia.testing import assert_close
 
+_ref = {
+    'cameras': (
+        'https://raw.githubusercontent.com/kornia/data/main/nerf/cameras.txt',
+        'aee6dbd448be900a0d4de85d08914e47a84f025e16f8498cbf9f7bfc7eff09a6',
+    ),
+    'images': (
+        'https://raw.githubusercontent.com/kornia/data/main/nerf/images.txt',
+        'e6b66a9b76d92e498697edece33be76ae69aa28a960bc895a59495763de61286',
+    ),
+}
+
+
+def _get_data(url: str, sha256: str) -> str:
+    req = urllib.request.Request(url)
+    with urllib.request.urlopen(req) as response:  # nosec
```

Comment:
`# nosec` because https://bandit.readthedocs.io/en/latest/blacklists/blacklist_calls.html#b310-urllib-urlopen -- but since it is from kornia repository, I don't see a problem, for extra precaution I added a sha256 check